src/modules/user/repository/user.repository.ts

```diff
@@ -48,11 +48,12 @@ export class UserRepository extends Repository<UserEntity> {
 
   async updateUser(id: number, data: UpdateUserDto) {
     const user = await this.findOne(id).catch(handleError);
+    data.email ? (user.email = data.email) : user.email;
+    data.name ? (user.name = data.name) : user.name;
+    data.password ? (user.password = data.password) : user.password;
+    data.type ? (user.type = data.type) : user.type;
```

Comment:
Quando for fazer update pode passar **data** diretamente para o método update.

Reply:
Fiz um novo commit utilizando o método Update para salvar os dados. Para manter o padrão de retorno, mantive o findOne, teria uma forma mais eficiente de fazer?